arda/js/contexts/app.js

```diff
@@ -0,0 +1,39 @@
+import Arda from 'arda'
+import AppComponent from '../components/App.jsx'
+
+export default class ArdaContext extends Arda.Context {
+    get subscribers() { return [require('./subscriber')]; }
+    get component() { return require('../components/App.jsx'); }
```

Comment:
@mizchi why `require` here? Isn't better and cleaner to make `import`s in the top?